dspace/config/modules/orcid.cfg

```diff
@@ -0,0 +1,130 @@
+orcid.mapping.funding.end-date = oairecerif.funding.endDate
+##orcid.mapping.funding.contributors syntax is <metadatafield>::<type>
+orcid.mapping.funding.contributors = project.investigator::lead
+orcid.mapping.funding.organization-relationship-type = isOrgUnitOfProject
+orcid.mapping.funding.amount = oairecerif.amount
+orcid.mapping.funding.amount.currency = oairecerif.amount.currency
+orcid.mapping.funding.amount.currency.converter = mapConverterDSpaceToOrcidAmountCurrency
+
+### Other names mapping ###
+orcid.mapping.other-names = person.name.variant
+orcid.mapping.other-names = person.name.translated
+
+### Keywords mapping ###
+orcid.mapping.keywords = dc.subject
+
+### Country mapping ###
+orcid.mapping.country = person.country
+orcid.mapping.country.converter =
+
+### Person External ids mapping ###
+##orcid.mapping.person-external-ids syntax is <metadatafield>::<type>
+orcid.mapping.person-external-ids = person.identifier.scopus-author-id::SCOPUS
+orcid.mapping.person-external-ids = person.identifier.rid::RID
+
+### Researcher urls mapping ###
+orcid.mapping.researcher-urls = oairecerif.identifier.url
+
+### Organization mapping ###
+orcid.mapping.organization.title = organization.legalName
+orcid.mapping.organization.country = organization.address.addressCountry
+orcid.mapping.organization.city = organization.address.addressLocality
+
+##orcid.mapping.organization.identifiers syntax is <metadatafield>::<source>
+##Valid organization identifier source values are ${orcid.validation.organization.identifier-sources}
+orcid.mapping.organization.identifiers = organization.identifier.crossrefid::FUNDREF
+orcid.mapping.organization.identifiers = organization.identifier.rin::RINGGOLD
+orcid.mapping.organization.identifiers = organization.identifier.lei::LEI
+
+### Contributor mapping ###
+orcid.mapping.contributor.email = person.email
+orcid.mapping.contributor.orcid = person.identifier.orcid
+
+#------------------------------------------------------------------#
+#--------------ORCID OBJECTS VALIDATION CONFIGURATIONS-------------#
+#------------------------------------------------------------------#
+
+orcid.validation.affiliation.enabled = true
+orcid.validation.work.enabled = true
+orcid.validation.funding.enabled = true
+
+orcid.validation.organization.identifier-sources = RINGGOLD
+orcid.validation.organization.identifier-sources = GRID
+orcid.validation.organization.identifier-sources = FUNDREF
+orcid.validation.organization.identifier-sources = LEI
+
+#------------------------------------------------------------------#
+#---------------------ORCID BULK SYNCHRONIZATION-------------------#
+#------------------------------------------------------------------#
+
+orcid.bulk-synchronization.max-attempts = 5
+
+#------------------------------------------------------------------#
+#--------------------ORCID EXTERNAL DATA MAPPING-------------------#
+#------------------------------------------------------------------#
```

Comment:
Again, it'd be very helpful to comment this section better. What are we "mapping" here...what is the key & value?  It looks like the key might be a field from ORCID? and the value is the field in DSpace?

What do the "converter" settings mean/reference?  Are those the names of the property files in config/crosswalks?

Basically we have to give normal users more documentation here.  It's very unclear how to use these configurations.